server/boot/user.js

```diff
@@ -234,15 +235,6 @@ module.exports = function(app) {
     });
   }
 
-  function getEmailSignup(req, res) {
```

Comment:
Maybe comment out this part instead of deleting it? We might revert this in the future if we have a good captcha for signing up.